data/mappings/info_config.json

```diff
@@ -80,6 +80,17 @@
     "RGBLIGHT_SLEEP": {"info_key": "rgblight.sleep", "value_type": "bool"},
     "RGBLIGHT_SPLIT": {"info_key": "rgblight.split", "value_type": "bool"},
     "RGBW": {"info_key": "rgblight.rgbw", "value_type": "bool"},
+    "POINTING_DEVICE_CS_PIN": {"info_key": "pointing_device.cs_pin"},
+    "POINTING_DEVICE_INVERT_X": {"info_key": "pointing_device.invert_x", "value_type": "bool"},
+    "POINTING_DEVICE_INVERT_X_RIGHT": {"info_key": "split.pointing_device.right.invert_x", "value_type": "bool"},
+    "POINTING_DEVICE_INVERT_Y": {"info_key": "pointing_device.invert_y", "value_type": "bool"},
+    "POINTING_DEVICE_INVERT_Y_RIGHT": {"info_key": "split.pointing_device.right.invert_y", "value_type": "bool"},
+    "POINTING_DEVICE_MOTION_PIN": {"info_key": "pointing_device.motion_pin"},
+    "POINTING_DEVICE_SDIO_PIN": {"info_key": "pointing_device.sdio_pin"},
+    "POINTING_DEVICE_SCLK_PIN": {"info_key": "pointing_device.sclk_pin"},
```

Comment:
How about grouping the hardware pins into a new structure, e.g.,:
```
"pointing_device": {
  "pins": {
    "motion": ...,
    "sclk": ...,
    "sdio": ...,
    "cs": ...,
  }
}
```
I don't think there is currently a way for these to differ on the right side, but this could provide a clean way.
`gestures` is also a nice sub-structure to group these attributes together.
```